src/Cms/Media.php

```diff
@@ -117,11 +118,12 @@ public static function thumb($model, string $hash, string $filename)
 				return false;
 			}
 
-			if (is_string($model) === true) {
-				$source = $kirby->root('index') . '/' . $model . '/' . $options['filename'];
-			} else {
-				$source = $model->file($options['filename'])->root();
-			}
+			$source = match (true) {
+				is_string($model) === true
+					=> $kirby->root('index') . '/' . $model . '/' . $options['filename'],
+				default
+				=> $model->file($options['filename'])->root()
+			};
```

Comment:
And here.

Reply:
Same as above

Reply:
Yep, same. I guess it comes down to a general team decision unless you are not super passionate about it @lukasbestle :)